packages/css/src/checkbox/README.md

```diff
@@ -1 +1,27 @@
 # Checkbox
+
+De checkbox (in het Nederlands: selectievakje) wordt gebruikt om gebruikers een selectie te laten maken. Hij wordt ook gebruikt om gebruikers te laten weten dat ze akkoord gaan met specifieke voorwaarden.
+
+## Richtlijnen
+
+- Selectievakjes worden gebruikt voor 1 of meerdere keuzes die elkaar niet uitsluiten.
+- Elk selectievakje werkt onafhankelijk van andere selectievakjes in een lijst op hetzelfde hiërarchische niveau. Het aanvinken van een extra vakje heeft geen invloed op andere selecties in die lijst.
+- Je kunt een selectievakje wel gebruiken voor een bulkselectie of –deselectie van een hele lijst met selectievakjes, maar dan moet duidelijk zijn dat het bulkselectievakje op een ander hiërarchisch niveau zit dan de lijst die volledig wordt geselecteerd of gedeselecteerd.
+- Een groep selectievakjes kan worden gebruikt om gegevens op een pagina, in een menu of binnen een component te filteren.
+- Een selectievakje kan een antwoord op een gesloten vraag zijn, als er meerdere antwoorden mogelijk zijn.
+- Een lijst van selectievakjes is verticaal gestapeld om leesbaarheid en bruikbaarheid te bevorderen.
+- Een lijst van selectievakjes moet in een `fieldset` zitten, met een `legend` die beschrijft waar de lijst over gaat. Als de selectievakjes bijvoorbeeld worden gebruikt om antwoord op een vraag te krijgen, dan is de `legend` de vraag.
+
+## Checkbox labels
+
+Een label:
+
+- ... begint met een hoofdletter
+- ... heeft geen interpunctie aan het einde als het een enkele zin, woord of fragment is.
```

Comment:
`text-wrap: balance` could be a nice addition to the longer labels, it make the block of text more of a whole (preventing orphaned words on a new line) and thus more tightly connected to the checkbox.

Reply:
That does look better. There is a downside to using it for interactive elements though. The hit area stays the same size, so you get a nice block of text with (imo unexpected) clickable space next to it. Let's discuss this tomorrow as well.

Reply:
Nice feature to consider, but out of scope for this documentation PR.